### ISSUE TITLE
Add JCDSv2 for resources and data sources

### ISSUE THREAD

**Issue by ecanault** [Yohan460/terraform-provider-jamf#19]
Hi,

Announced in Jamf Pro 11, JCDS v2 will be available soon.
It would be nice to add API endpoint and methods when they will be available :-)

**Comment by w0de** [comment]
I agree, package resources backed by the cloud distribution API would be wonderful.
